### PR TITLE
Check in 242.py.

### PR DESCRIPTION
**Pull request by woke7**

### CODE REVIEW

File: 242.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from collections import defaultdict
+
+class Solution:
+    def isAnagram(self, s: str, t: str) -> bool:
+        table = defaultdict(lambda: 0)
+        
+        if len(s) != len(t):
```
[the remark this answers is not in the document]
Comment (woke7) [reply 242.py:7]: Hmm...I think we should still bail faster if that is doable. Performance of the code is also very important.

File: 242.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from collections import defaultdict
+
+class Solution:
+    def isAnagram(self, s: str, t: str) -> bool:
+        table = defaultdict(lambda: 0)
```
[the remark this answers is not in the document]
Comment (woke7) [reply 242.py:5]: I have a hard time coming up with a suitable naming for simple usage.

File: 242.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        for char in s:
+            table[char] += 1
+        
+        for char in t:
+            if char not in table:
+                return False
+
+            table[char] -= 1
+            
+        return all(value == 0 for value in table.values())
```
[the remark this answers is not in the document]
Comment (woke7) [reply 242.py:19]: I got what you are saying, but will this be considered cheating in an interview?

[the remark this answers is not in the document]
Comment (mujingw) [reply 242.py:19]: More realistically, straightforward problems like this one are not being asked during an interview these days.

File: 242.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mujingw) [reply 242.py:7]: The gain in performance might not be discernible. On top of that, if the majority of your input strings are with equal lengths, this step won't help much.

File: 242.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mujingw) [reply 242.py:5]: You can still name it as `char_count`, `count`, or something similar.

**Review by mujingw — APPROVED** [review]
LGTM

**Pull request merged**